hi-ml-histopathology/src/histopathology/utils/wsi_utils.py

```diff
@@ -3,7 +3,7 @@
 
 from typing import List
 from histopathology.utils.naming import SlideKey
-from torch.utils.data.dataloader import default_collate
+from health_ml.utils.bag_utils import multibag_collate
 
 
 def image_collate(batch: List[dict]) -> List[dict]:
```

Comment:
do we want to have a test for `image_collate` as codecov has some warnings

Reply:
Yes, just discussed that on Teams with Daniel. I will add a test for that.